src/components/amount-buttons.js

```diff
@@ -85,13 +85,13 @@ var AmountOtherButton = React.createClass({
     var inputValue = e.currentTarget.value;
     var amount = "";
 
-    if (/^[\d]*[\.]?\d{0,2}$/.test(inputValue)) {
+    if (/^[\d]*[.]?\d{0,2}$/.test(inputValue)) {
```

Comment:
actual code change

Reply:
Can you explain the code change? The origin RE matches an optional `\` or `.` in the second character class, was this a bug?

Reply:
don't think the original RE matches the optional `\`. I think for it to do that it would need to be `\\` since `\` is an escape character inside the `[ ]`. So theoretically, both before and after this change, the regex should do the same thing. I'm checking in with @ScottDowne though.